navigation/ca_cartographer/config/ca_2d.lua

```diff
@@ -0,0 +1,53 @@
+include "map_builder.lua"
+include "trajectory_builder.lua"
+
+options = {
+  map_builder = MAP_BUILDER,
+  trajectory_builder = TRAJECTORY_BUILDER,
+  map_frame = "map",
+  tracking_frame = "create1_tf/imu_link",
```

Comment:
This will fail with any other robot. Is there a way to test change it based on an environment variable?

Reply:
Added ID as environment dynamic value.